testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/realm/ClientPoliciesTest.java

```diff
@@ -0,0 +1,396 @@
+import org.jboss.arquillian.graphene.page.Page;
+import org.junit.After;
+import org.junit.Test;
+import org.keycloak.authentication.authenticators.client.ClientIdAndSecretAuthenticator;
+import org.keycloak.authentication.authenticators.client.JWTClientAuthenticator;
+import org.keycloak.common.Profile;
+import org.keycloak.representations.idm.ClientPoliciesRepresentation;
+import org.keycloak.representations.idm.ClientProfilesRepresentation;
+import org.keycloak.services.clientpolicy.condition.ClientAccessTypeConditionFactory;
+import org.keycloak.services.clientpolicy.executor.HolderOfKeyEnforcerExecutorFactory;
+import org.keycloak.services.clientpolicy.executor.SecureClientAuthenticatorExecutorFactory;
+import org.keycloak.testsuite.arquillian.annotation.EnableFeature;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.ClientPolicies;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.ClientPoliciesJson;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.ClientPolicy;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.ClientProfile;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.ClientProfiles;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.ClientProfilesJson;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.Condition;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.CreateClientPolicy;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.CreateClientProfile;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.CreateCondition;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.CreateExecutor;
+import org.keycloak.testsuite.console.page.realm.clientpolicies.Executor;
+import org.keycloak.testsuite.util.ClientPoliciesUtil.ClientPoliciesBuilder;
+import org.keycloak.testsuite.util.ClientPoliciesUtil.ClientPolicyBuilder;
+import org.keycloak.testsuite.util.ClientPoliciesUtil.ClientProfileBuilder;
+import org.keycloak.testsuite.util.ClientPoliciesUtil.ClientProfilesBuilder;
+import org.keycloak.util.JsonSerialization;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.keycloak.testsuite.util.ClientPoliciesUtil.createClientAccessTypeConditionConfig;
+import static org.keycloak.testsuite.util.ClientPoliciesUtil.createHolderOfKeyEnforceExecutorConfig;
+import static org.keycloak.testsuite.util.ClientPoliciesUtil.createSecureClientAuthenticatorExecutorConfig;
+import static org.keycloak.testsuite.util.UIUtils.refreshPageAndWaitForLoad;
+
+/**
+ * @author Vaclav Muzikar <vmuzikar@redhat.com>
+ */
+@EnableFeature(Profile.Feature.CLIENT_POLICIES)
+public class ClientPoliciesTest extends AbstractRealmTest {
+    private static final String GLOBAL_PROFILE = "global-default-profile";
```

Comment:
This profile was a placeholder for default profile, which was removed in https://github.com/keycloak/keycloak/pull/8093 and replaced with the proper FAPI profiles. So it seems this will need some changes to not be dependent on "global-default-profile"

Reply:
Done.